libs/template/package.json

```diff
@@ -43,9 +43,9 @@
     "url": "https://github.com/rx-angular/rx-angular.git"
   },
   "peerDependencies": {
-    "@angular/core": "^17.0.0",
+    "@angular/core": "^18.0.0",
     "rxjs": "^6.5.3 || ^7.4.0",
```

Comment:
I can't find the information on the website but I'm wondering if Angular v18 changed its compat range with RxJS?

Reply:
doesn't look like it https://github.com/angular/angular/blob/main/packages/core/package.json